tools/make/lib/lint/typescript/Makefile

```diff
@@ -32,19 +32,18 @@ include $(TOOLS_MAKE_LIB_DIR)/lint/typescript/dtslint.mk
 # -   This rule supports the environment variables supported by each context-specific (`lint-typescript-<context>`) prerequisite.
 # -   This rule is useful when wanting to glob for files, irrespective of context, for a particular package in order to lint all contained TypeScript files.
 #
-# @param {string} [TYPESCRIPT_SOURCES_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
-# @param {string} [TYPESCRIPT_TESTS_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
+# @param {string} [SOURCES_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
```

Comment:
The change from `TYPESCRIPT_SOURCES_FILTER` to `SOURCES_FILTER` is incorrect. To see how these environment variables interact, please review https://github.com/stdlib-js/stdlib/blob/develop/tools/make/lib/ls/Makefile. `SOURCES_FILTER` is intended to be used with `SOURCES_PATTERN` which resolves JavaScript files by default. This change should be reverted here and elsewhere.